<launch>

    <!-- <node name="realsense_processing" pkg="single_camera_demo" type="realsense_processing.py"> -->
    <node pkg="tf2_ros" type="static_transform_publisher" name="base_to_aruco_broadcaster"
        args="0.21 0.0 -0.04   0.0 0.0 0.7071 0.7071  base_link aruco_marker_frame" />

    <!-- <node pkg="tf2_ros" type="static_transform_publisher" name="base_to_camera_tf" 
     args="0.024 -0.320 0.642 -0.413 0.422 0.570 0.571 aruco_marker_frame camera_link"/> -->
    <node pkg="tf2_ros" type="static_transform_publisher" name="base_to_camera_tf" 
     args="0.6150792305377963 0.02226115583018931 0.08599622850224437 0.4047853186694866 -0.39513742850322936 -0.5814433363751594 0.5847554232073617 camera_link aruco_marker_frame"/>
    <include file="$(find openDay_demo)/launch/setup_robot.launch"/>
    <include file="$(find openDay_demo)/launch/setup_435_camera.launch"/>
    <include file="$(find openDay_demo)/launch/aruco_ros_single.launch"/>
    
    <node name="camera_broadcast" pkg="openDay_demo" type="publish_camera.py" output="screen" required="true">
        <param name="parent_frame" value="base_link"/>
        <param name="child_frame" value="camera_link"/>
    </node>

    
    
    <!-- <include file="$(find single_camera_demo)/launch/publish_camera.launch"/> -->
    <include file="$(find openDay_demo)/launch/single_camera_processing.launch"/>
    <!-- <include file="$(find ggcnn)/launch/ggcnn_service.launch"/> -->
    



    <node name="detect_grasps" pkg="gpd_ros" type="detect_grasps" output="screen">
        <param name="cloud_type" value="0" />
        <param name="cloud_topic" value="/cloud_to_get_grasps_on" />
        <param name="config_file" value="/home/riot/kinova_gen3_lite/src/openDay_demo/config/ros_eigen_params_left.cfg" />
        <param name="rviz_topic" value="plot_grasps" />
    </node>


</launch>